backends/events/sdl/sdl-events.h

```diff
@@ -236,7 +236,11 @@ class SdlEventSource : public Common::EventSource {
 	};
 
 	struct TouchFinger {
+#if SDL_VERSION_ATLEAST(3, 0, 0)
```

Comment:
I would add a TODO here to say that SDL2 should be migrated too when the new changes are tested enough.

Reply:
I looked again and couldnt find anything about SDL_FingerID in SDL2 being 0 when invalid. Its only documented for SDL_TouchID. If thats the case, we might never switch. 

@rsn8887 do you remember by any chance why you picked -1 and not 0 for invalid/lifted FingerIDs?